.github/workflows/current.yml

```diff
@@ -0,0 +1,53 @@
+name: Test on current Node.js
+
+on:
+  schedule:
+    - cron: '00 00 * * Mon'
+  workflow_dispatch:
```

Comment:
This should let us manually run it whenever we want, but I don't see a way to start a new workflow from a branch; I think it has to be on `main` already.